keyboards/mode/eighty/m80s/m80s.h

```diff
@@ -0,0 +1,32 @@
+ /* Copyright 2020 Álvaro "Gondolindrim" Volpato <gondolindrim@acheronproject.com>
+  * 
+  * This program is free software: you can redistribute it and/or modify 
+  * it under the terms of the GNU General Public License as published by 
+  * the Free Software Foundation, either version 2 of the License, or 
+  * (at your option) any later version. 
+  * 
+  * This program is distributed in the hope that it will be useful, 
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  */
+
+#pragma once
+
+#include "eighty.h"
+
+#define LAYOUT_eighty_m80s( \
```

Comment:
Would `hotswap` and `soldered` be better names for the layout macros and subfolders?

Reply:
I used m80h and m80s because the PCBs themselves are named like that, so it made sense. To be honest I don't see a reason to change, it's already pretty straightforward